src/components/pages/RegistrationLinkSentPage.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { useQuery } from 'hooks/query-params';
+
+function RegistrationLinkSent() {
+  const queryParams  = useQuery();
+  const email = queryParams.get('email');
+
+  return (
+    <div className="container mx-auto text-center">
```

Comment:
Comment (non-blocking):
This top level div can be a section element, instead of a div